src/t_zset.c

```diff
@@ -2257,26 +2257,14 @@ int zuiFind(zsetopsrc *op, zsetopval *val, double *score) {
         return 0;
 
     if (op->type == OBJ_SET) {
-        if (op->encoding == OBJ_ENCODING_INTSET) {
-            if (zuiLongLongFromValue(val) &&
-                intsetFind(op->subject->ptr,val->ell))
-            {
-                *score = 1.0;
-                return 1;
-            } else {
-                return 0;
-            }
-        } else if (op->encoding == OBJ_ENCODING_HT) {
-            dict *ht = op->subject->ptr;
-            zuiSdsFromValue(val);
-            if (dictFind(ht,val->ele) != NULL) {
-                *score = 1.0;
-                return 1;
-            } else {
-                return 0;
-            }
+        zuiSdsFromValue(val);
```

Comment:
Actually when I think about it, `val` sometimes represents an integer, sometimes an sds and sometimes a C string. We can avoid converting to sds if we use `setTypeIsMemberAux`. Maybe something like this? (Or maybe something even more smart is possible, I'm not sure.)

```C
char *str = val->ele ? val->ele : val->estr;
size_t len = val->ele ? sdslen(val->ele) : val->elen;
if (setTypeIsMemberAux(op->subject, str, len, val->ell, val->ele != NULL)) { ... }

Reply:
Good. It seems to work. :-) I forgot that setTypeIsMemberAux was internal. I think it's fine to make it public.